### PR TITLE
Update Password to have no digits generated

### PR DESCRIPTION
**Pull request by AndreiH**

### CODE REVIEW

File: fxapom/fxapom.py [hunk 1]
```diff
@@ -36,6 +38,8 @@ def __init__(self, mozwebqa=None, use_prod=True):
     def create_account(self):
         self.account = TestEmailAccount()
         client = Client(self.fxa_url)
+        print self.account.email
+        print self.password
```
Comment (bobsilverberg) [inline fxapom/fxapom.py]: I don't think we should be outputting the password to the console, especially as we plan on making Jenkins public in the near future.

File: fxapom/fxapom.py [hunk 2]
```diff
@@ -36,6 +38,8 @@ def __init__(self, mozwebqa=None, use_prod=True):
     def create_account(self):
         self.account = TestEmailAccount()
         client = Client(self.fxa_url)
+        print self.account.email
```
Comment (bobsilverberg) [inline fxapom/fxapom.py]: I suggest that a log message should contain something to indicate what it is. Just outputting the email address and nothing else won't be easy to find or understand in the output. Perhaps something like `print 'fxapom created an account for email: %s at %s' % (self.account.email, now())`

File: fxapom/fxapom.py (hunk 1 above)
Comment (AndreiH) [inline fxapom/fxapom.py]: ups, ok.  So just keep the email?

**Comment by bobsilverberg** [comment]
Thanks for this @AndreiH. I added a couple of comments.

File: fxapom/fxapom.py (hunk 2 above)
Comment (AndreiH) [inline fxapom/fxapom.py]: :+1:

**Comment by AndreiH** [comment]
Updated the PR @bobsilverberg !

**Comment by bobsilverberg** [comment]
Looks good. Thanks @AndreiH!

**Pull request merged**